### ISSUE TITLE
Error coming in tensorflow_backend.py file under pool3d(...) function.

### ISSUE THREAD

**Issue by alok2490** [keras-team/keras#10141]
I'm using below line of code while designing my network but it is causing error.

x = MaxPool3D((2,2,1),strides=(2,2,1),padding='same')(x)

I feel that there is a problem in file tensorflw_backend.py under pool3d funtion : 
###########################################
if pool_mode == 'max':
        x = tf.nn.max_pool3d(x, pool_size, strides,
                             padding=padding,
                             data_format=tf_data_format)
    
############################################

problem here is coming because we are passing data_format to gen_nn_ops.py file which is machine generated where it is not expecting any data_format parameter as such and therefore causing error while running. 

Kindly check at your end once and respond if something is wrong at my end.

Thanks in advance.

Regards,
Alok.
